find_unavailable_instance_types.py

```diff
@@ -52,24 +55,26 @@ def get_spot_history(region, types):
                 types_in_azs[az] = set()
             types_in_azs[az].add(t)
 
-    return { az: list(types) for az, types in types_in_azs.items() }
+    return {az: list(types) for az, types in types_in_azs.items()}
```

Comment:
yuck! how is this better without a space after `{`?

Reply:
it's better in the sense it's the canonical style guide for python :-)

Reply:
[![Whitespace.  The final frontier.](https://user-images.githubusercontent.com/7802932/39092722-e9677ce0-461c-11e8-9d1c-fb06ae50d50a.png)](https://imgflip.com/i/28uxow)